src/webgpu/gpu_test.ts

```diff
@@ -526,11 +527,18 @@ export class GPUTestBase extends Fixture<GPUTestSubcaseBatchState> {
    * Skips test if any format is not supported.
    */
   skipIfTextureFormatNotSupported(...formats: (GPUTextureFormat | undefined)[]) {
-    if (isCompatibilityDevice(this.device)) {
-      for (const format of formats) {
-        if (format === 'bgra8unorm-srgb') {
-          this.skip(`texture format '${format} is not supported`);
+    for (const format of formats) {
+      if (format) {
+        if (isCompatibilityDevice(this.device)) {
+          if (format === 'bgra8unorm-srgb') {
```

Comment:
nit: prefer ifs in order

```
if (format === 'bgra8unorm-srgb') {
  if (isCompatibilityDevice(this.device)) {
```

as it only enters the if branch once

Reply:
done